scipy-1.0/references.bib

```diff
@@ -1510,3 +1510,22 @@ @online{asvdocs
 url = {https://asv.readthedocs.io/en/stable/index.html},
 urldate = {2019-1-22}
 }
+
+% this reference manual has been cited
+% > 300 times, so perhaps suitable here
+% for CPython
+@online{cpython-refman,
+author = {{Guido van Rossum}},
```

Comment:
There's an editor on this whitepaper too, so not sure if "online" is the correct category or not. An argument may also be made that we should just cite the more recent community-generated manuals in the docs.

Reply:
I'll review citation details all at once at the end.